### ISSUE TITLE
Email: Improve affordances for selecting and purchasing an email plan

### ISSUE THREAD

**Issue by matt-west** [Automattic/wp-calypso#95866]
### Quick summary

When selecting an email plan, the user must click the "select" button which expands the plan container, but no active state is shown on the selected container. The purchase action is also placed within the plan container, whereas we usually place actions outside of the direction selection.

We should consider using customised radio buttons for selecting a plan and placing the purchase actions globally on the page.

![Image](https://github.com/user-attachments/assets/72010ca3-4c71-4264-b00b-e87e6125ef5d)

### Steps to reproduce

Go to WP Admin -> Hosting -> Emails.

### What you expected to happen

* We should use radio buttons/familiar blue outlines to highlight the selected plan.
* The Purchase should be placed globally on the page rather than within the selected plan container.

### What actually happened

See summary.

### Impact

One

### Available workarounds?

No but the platform is still usable

**Comment by Robertght** [comment]
I agree the section can be clearer and I came up with this to visually represent it and create a clear distinction between the expanded bit. It's a suggestion:

![Image](https://github.com/user-attachments/assets/931bb6e7-f038-4766-9a2c-cb83d4a5d070)

Keeping this as a low priority for now.

cc @Automattic/yolo if you're still focusing on this product bit

**Comment by fredrikekelund** [comment]
I think what @matt-west suggests is a slightly larger overhaul than just applying more active state to the selected container, so I added the `Needs Design` label here. 

I should also mention that team Nexus is working on the email management pages now to fit them into the "all domains" view on `/domains/manage`. cc @donnapep for any potential input around this.

**Comment by donnapep** [comment]
Thanks for the ping! I'll defer to @Imran92 on this, who is leading that particular project.
